packages/esm-amd-loader/test/src/suite.ts

```diff
@@ -268,19 +268,25 @@ suite('dynamic require', () => {
   });
 
   test('calls error callback only once on multiple 404s', (done) => {
-    let numErrors = 0;
+    let num404s = 0;
+    let numCallbackCalls = 0;
+
+    window.addEventListener('error', on404, true);
```

Comment:
Hm, I don't think this is the expected behavior. `require` should not throw a global error, it should only call the failure callback shouldn't it?

Reply:
Oh, is this the browser doing this? Emitting a window error?

Reply:
Yeah, you can capture the browser network errors by adding the capture phase listener (the true).